gadget/gadget.go

```diff
@@ -1725,7 +1725,7 @@ func checkCompatibleSchema(old, new *Volume) error {
 // LaidOutVolumesFromGadget takes gadget volumes, gadget and kernel rootdirs
 // and lays out the partitions on all volumes as specified. It returns the
 // volumes mentioned in the gadget.yaml and their laid out representations.
-func LaidOutVolumesFromGadget(vols map[string]*Volume, gadgetRoot, kernelRoot string, encType secboot.EncryptionType, volToGadgetToDiskStruct map[string]map[int]*OnDiskStructure) (all map[string]*LaidOutVolume, err error) {
+func LaidOutVolumesFromGadget(vols map[string]*Volume, gadgetRoot, kernelRoot string, encType device.EncryptionType, volToGadgetToDiskStruct map[string]map[int]*OnDiskStructure) (all map[string]*LaidOutVolume, err error) {
```

Comment:
since we're already modifying the code, why not do some formatting improvements as well
```suggestion
func LaidOutVolumesFromGadget(
	vols map[string]*Volume, 
	gadgetRoot, kernelRoot string, 
	encType device.EncryptionType,
	volToGadgetToDiskStruct map[string]map[int]*OnDiskStructure,
) (all map[string]*LaidOutVolume, err error) {
```